app/javascript/components/admin/UserModal.js

```diff
@@ -18,6 +18,7 @@ class UserModal extends React.Component {
       sorted_jurisdiction_paths: _.values(this.props.jurisdiction_paths).sort((a, b) => a.localeCompare(b)),
       jurisdiction_path: this.props.initialUserData.jurisdiction_path || this.props.jurisdiction_paths[0],
       roleTitle: this.props.initialUserData.role_title || this.props.roles[0],
+      roles: this.props.roles,
```

Comment:
you probably don't need to set this in state since it just stays constant

Reply:
Ditto! [ea9253d6956658ca2db9763a9140c370e806cb31](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/ea9253d6956658ca2db9763a9140c370e806cb31)